rollup.config.js

```diff
@@ -0,0 +1,34 @@
+import resolve from "rollup-plugin-node-resolve";
+import commonjs from "rollup-plugin-commonjs";
+import pkg from "./package.json";
+
+export default [
+  // browser-friendly UMD build
+  {
+    input: "src/index.js",
+    output: {
+      name: "cryptoformat",
+      file: pkg.browser,
+      format: "umd"
+    },
+    plugins: [
+      resolve(), // so Rollup can find `dependencies`
+      commonjs() // so Rollup can convert `dependencies` to an ES module
+    ]
```

Comment:
Just curious, why plugins only on umd?

Reply:
Thanks for the question, just updated to standardise all the umd, esm and cjs. (Currently cryptoformat does not have any dependencies yet)